index.html

```diff
@@ -883,9 +827,7 @@ <h3>Role Mapping Table</h3>
 								</td>
 								<td><p>Expose <code>ROLE_LINK</code> and object attribute
 											<code>xml-roles:doc-noteref</code>. </p></td>
-								<td>AXRole: <code>AXLink</code><br /> AXSubrole: <code>&lt;nil&gt;</code><br />
-									AXRoleDescription: <code>'link'</code>
-								</td>
+								<td>AXRole: <code>AXLink</code><br /> AXSubrole: <code>&lt;nil&gt;</code></td>
```

Comment:
We had agreed at TPAC to keep the link subroles, among others. Still investigating a way to add additional clarifiers, but this PR should probably be moved back to Draft until the corresponding ARIA issue is settled.

Reply:
One of the proposals discussed was to expose the applied aria role through the API... Similar to how the DOM id is exposed.